package.json

```diff
@@ -36,7 +36,7 @@
     "docker:clean": "docker rmi force force:builder force:electron",
     "jest": "node_modules/.bin/jest --config jest.config.js",
     "jest:debug": "node --inspect node_modules/.bin/jest --runInBand",
-    "jest:terse": "TERSE_RTL_OUTPUT=true jest",
+    "jest-enzyme": "node_modules/.bin/jest --config jest.legacy.config.js",
```

Comment:
**question:** Intentionally dropping the `terse` package script?

Reply:
Yah, that was a silly "me problem" 🤦 And don't know what i was thinking there

Reply:
Ie: 99% of the time its helpful. I ran into the 1% and caused a lil fuss and having this extra script just complicates things.